src/orchestrator.rs

```diff
@@ -164,16 +178,23 @@ async fn chunk_and_detect(
     text: String,
     masks: Option<&[(usize, usize)]>,
 ) -> Result<Vec<TokenClassificationResult>, Error> {
-    // TODO: propogate errors
     // Apply masks
     let text_with_offsets = masks
         .map(|masks| apply_masks(&text, masks))
         .unwrap_or(vec![(0, text)]);
     // Create a list of required chunkers
     let chunker_ids = detectors
         .keys()
-        .map(|detector_id| ctx.config.get_chunker_id(detector_id))
-        .collect::<Vec<_>>();
+        .map(|detector_id| {
+            let chunker_id =
+                ctx.config
+                    .get_chunker_id(detector_id)
```

Comment:
Same answer:
Yes, it will be checked during config validation, so in theory this error should never be returned here. We should still be safe and use ok_or_else() here rather than unwrap() though. Especially now, considering config validation isn't implemented yet.